examples/bickley_jet_cmc_cg.jl

```diff
@@ -0,0 +1,106 @@
+push!(LOAD_PATH, joinpath(@__DIR__, "..", ".."))
+
+using ClimaCore.Geometry, LinearAlgebra, UnPack
+import ClimaCore: Fields, Domains, Topologies, Meshes, Spaces
+import ClimaCore.Operators
+import ClimaCore.Geometry
+using LinearAlgebra, IntervalSets
+using OrdinaryDiffEq: ODEProblem, solve, SSPRK33
+
+using Logging: global_logger
+using TerminalLoggers: TerminalLogger
+global_logger(TerminalLogger())
+
+# set up boilerplate
+include("../src/backends/WIP_backends.jl")
+include("../src/interface/WIP_domains.jl")
+include("../src/interface/WIP_physics.jl")
+include("../src/interface/WIP_boundary_conditions.jl")
+include("../src/interface/WIP_models.jl")
+include("../src/interface/WIP_timesteppers.jl")
+include("../src/interface/WIP_simulations.jl")
+
+# set up parameters
+const parameters = (
+    ϵ = 0.1,   # perturbation size for initial condition
+    l = 0.5,   # Gaussian width
+    k = 0.5,   # Sinusoidal wavenumber
+    ρ₀ = 1.0,  # reference density
+    c = 2,
+    g = 10,
+    D₄ = 1e-4, # hyperdiffusion coefficient
+)
+
+# set up domain
+domain = Rectangle(
+    xlim = -2π..2π, 
+    ylim = -2π..2π, 
+    nelements = (16, 16), 
+    npolynomial = 4, 
+    periodic = (true, true)
+)
+
+# set up initial condition
+function init_state(x, p)
```

Comment:
since parameters are const, can we directly using parameters in the function?

Reply:
Not sure what you mean :)?